docs/yaml/elements/precinct.yaml

```diff
@@ -31,9 +31,7 @@ tags:
   description: Links to an :ref:`$$$-electoral-district` (e.g., congressional district,
     state house district, school board district) to which the precinct belongs. **Highly
     Recommended** if candidate information is to be provided. Multiple allowed and
-    recommended to specify the geography of multiple electoral districts. If an electoral
-    district splits a precinct, use the `PrecinctSplitName` object and do not specify
-    that particular electoral district in this object.
+    recommended to specify the geography of multiple electoral districts.
```

Comment:
It's actually correct behavior (I believe) to specify all the electoral district IDs for each split, and not just the ones where they differ. @indymag?